test-framework/sudo-compliance-tests/src/visudo/flag_file.rs

```diff
@@ -192,7 +192,7 @@ echo "$@" > {LOGS_PATH}"#
 #[test]
 fn regular_user_can_create_file() -> Result<()> {
     let env = Env("")
-        .file(DEFAULT_EDITOR, TextFile(EDITOR_TRUE).chmod("111"))
+        .file(DEFAULT_EDITOR, TextFile(EDITOR_DUMMY).chmod("755"))
```

Comment:
It seems like this test was previously passing only because visudo created the target file even when the editor fails to run. This is no longer true.